_inc/client/components/upgrade-link/index.jsx

```diff
@@ -16,14 +17,39 @@ import { getUpgradeUrl } from 'state/initial-state';
 class UpgradeLink extends PureComponent {
 	static propTypes = {
 		source: PropTypes.string.isRequired,
+		target: PropTypes.string.isRequired,
+		feature: PropTypes.string.isRequired,
 
 		// Connected
 		upgradeUrl: PropTypes.string.isRequired,
 	};
 
+	constructor() {
+		super( ...arguments );
+
+		this.trackClick = this.trackClick.bind( this );
```

Comment:
This binding could be dropped if we dinfed `trackClick` as a class property like this instead:

```
trackClick = () => {
};
```